src/platforms/hubspot/actions/deals/update.ts

```diff
@@ -0,0 +1,42 @@
+import { transformDeal } from '@/platforms/hubspot/actions/mappers';
+import { action } from '@/sdk';
+import * as custom from '@/sdk/validators';
+import { z } from 'zod';
+import client from '../../client';
+
+export default action(
+  'deals-update',
+  {
+    operation: 'update',
+    resource: 'deals',
+    mutation: true,
+    schema: z
+      .object({
+        closeDate: custom.date(),
+        dealName: z.string().optional(),
+        amount: z.string().optional(),
+        hsDealStageProbability: z.string().optional(),
+        dealStage: z.string().optional(),
+      })
+      .partial()
+      .extend({
+        id: z.string(),
+      }),
```

Comment:
how come you're calling .partial() when several fields are already optional?

Reply:
Ah yeah, let me remove the optionals. I was having this match up with our unified requirements, but hubspot lets you create objects without any properties. So I just updated all the inputs for creates/updates to be partial